src/api/app/models/scm_webhook.rb

```diff
@@ -72,6 +76,10 @@ def gitlab_push_event?
     @payload[:scm] == 'gitlab' && @payload[:event] == 'Push Hook'
   end
 
+  def github_ignored_push_event?
+    github_push_event? && @payload[:deleted]
+  end
+
```

Comment:
[nitpick] Please move this method to the bottom, where the rest of `*ignored*` methods are.

Reply:
And let's follow the same name pattern:
```suggestion
  def github_ignored_github_push_event?
    github_push_event? && @payload[:deleted]
  end

```

Reply:
done